src/components/about-us/section-05/list.js

```diff
@@ -0,0 +1,364 @@
+        width: 100%;
+        height: ${props => props.unfold ? '0' : '100%'};
+        z-index: calc(${defaultZIndex} - 1);
+        transition: height .25s;      
+      }
+    }
+    &:last-child {
+      display: ${props => !props.unfold && props.isOdd ? 'none' : 'block'};
+    }  
+  `}
+`
+
+export default class List extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.yearContent = []
+  }
+  _getMonth = (month) => {
+    switch(month) {
+      case 1:  
+        return '一月'
+      case 2:
+        return '二月'
+      case 3:
+        return '三月'
+      case 4:
+        return '四月'
+      case 5:
+        return '五月'
+      case 6:
+        return '六月'
+      case 7:
+        return '七月'
+      case 8:
+        return '八月'
+      case 9:
+        return '九月'
+      case 10:
+        return '十月'
+      case 11:
+        return '十一月'
+      case 12:
+        return '十二月'
+    }
+  }
+
+  _getDate = (date) => {
+    let dateString = date.toString()
+    if (dateString.length < 2) {
+      return '0' + dateString
+    }
+    return dateString
+  }
+  
+  /**
+  + * Opens the valid input link in a new window or tab
+  + * @param {String} href
+  + */
+  _openLink = (href) => {
+    if (!href) return
+    window.open(href, '_blank')
+  }
+
+  componentDidMount() {
+    const yearContentHeight = this.yearList.map((year, index) => ReactDOM.findDOMNode(this.yearContent[index]).getBoundingClientRect().height)
+    this.props.getYearContentHeight(yearContentHeight)
+  }
+
+  render() {
+    const { unfoldArray, sortedData, sortedDataGroupByYear, foldAndUnfold } = this.props
+    this.yearList = Object.keys(sortedDataGroupByYear)
+    const sortedDataGroupByYearMonth = Object.values(_.groupBy(sortedData, record => record.year)).map((dataEachYear) => {
+      return _.groupBy(dataEachYear, data => data.month)
+    })
+    const groupedData = this.yearList.reduce((previous, year, index) => {
+      previous[year] = sortedDataGroupByYearMonth[index]
+      return previous
+    }, {})
+    const Records = (year, indexOfUnfoldArray) => {
+      let isOdd = sortedDataGroupByYear[year].length % 2 === 1
+      return (
+        <Acomplishments
+          unfold={unfoldArray[indexOfUnfoldArray]}
+        >
+          <OnlyDisplayOnMobile>
+            {
+              sortedDataGroupByYear[year].map((record, index) => {
+                return(
+                    <Acomplishment
+                      key={index}
+                      unfold={unfoldArray[indexOfUnfoldArray]}
+                      isOdd={isOdd}
+                    >
+                      <MonthLabel>
+                        <p>{this._getMonth(record.month)}</p>
+                      </MonthLabel>
+                      <Record>
+                        <p>
+                          <span>{this._getDate(record.date)}</span>
+                        </p>
+                        <p onClick={() => this._openLink(record.link)}>{record.text.chinese}</p>
```

Comment:
How do user know whether `Record` could be clicked since the appearance of records are the same ?

Reply:
The link will be removed, since it could be easily clicked when user are just swiping.